src/commands/svc/list.ts

```diff
@@ -0,0 +1,35 @@
+import { CliUx, Command, Flags } from '@oclif/core'
+import { BundleService } from '../../services/bundle-service'
+import { SvcService } from '../../services/svc-service'
+
+export default class List extends Command {
+  static description = 'List auxiliary services'
+
+  static examples = ['<%= config.bin %> <%= command.id %>']
+
+  static args = []
+
+  static flags = {
+    available: Flags.boolean({ description: 'List all available services' })
+  }
+
+  public async run(): Promise<void> {
+    BundleService.verifyBundleInitialized(process.cwd())
+
+    const { flags } = await this.parse(List)
+
+    const svcService: SvcService = new SvcService(process.cwd())
+    const services: string[] = flags.available
+      ? svcService.getAvailableServices()
+      : svcService.getActiveServices()
+    const serviceData = services.map(service => ({ service }))
+
+    const columns = {
+      service: {
+        header: flags.available ? 'Available Services' : 'Active Services'
+      }
```

Comment:
After listing the available services I would like to add a message like this: "Hint: you can add an auxiliary service just dropping a Docker Compose file in the svc folder (e.g. svc/my-service.yml)"

Reply:
log added at the end of table listing